mono/mini/mini-llvm.c

```diff
@@ -8261,6 +8261,10 @@ emit_method_inner (EmitContext *ctx)
 
 	mono_llvm_add_func_attr (method, LLVM_ATTR_UW_TABLE);
 
+	if (cfg->disable_omit_fp) {
+                mono_llvm_add_func_attr_nv (method, "no-frame-pointer-elim", "true");
+        }
```

Comment:
Formatting: Mono uses tabs and doesn't use braces to demarcate single-statement blocks.